extensions/Worker.Extensions.ServiceBus/src/ServiceBusMessageActionsConverter.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.Azure.Functions.Worker.Converters;
+using Microsoft.Azure.ServiceBus.Grpc;
+
+namespace Microsoft.Azure.Functions.Worker
+{
+    /// <summary>
+    /// Converter to bind to <see cref="ServiceBusMessageActions" /> type parameter.
+    /// </summary>
+    internal class ServiceBusMessageActionsConverter : IInputConverter
+    {
+        private readonly Settlement.SettlementClient _settlement;
+
+        public ServiceBusMessageActionsConverter(Settlement.SettlementClient settlement)
+        {
+            _settlement = settlement;
+        }
+
+        public ValueTask<ConversionResult> ConvertAsync(ConverterContext context)
+        {
+            try
+            {
+                return new ValueTask<ConversionResult>(ConversionResult.Success(new ServiceBusMessageActions(_settlement)));
+            }
+            catch (Exception exception)
```

Comment:
Just a note to myself: is this expected for all input converters to add such trivial exception handling? If line 27 ever failed it would mean there is something seriously wrong. Is it better to let that exception bubble up?

Reply:
In this particular case, I would expect is to let this exception bubble up.

Reply:
My understanding was that catching and returning a failed result was the recommended pattern. This is what we do for the other SDK type converters. 
/cc @liliankasem

Reply:
Yeah I'm pretty sure every converter will fail with the exception:

e.g. cosmos: https://github.com/Azure/azure-functions-dotnet-worker/blob/7f4e1a1925cb11ec341d02da094dd25908a49e89/extensions/Worker.Extensions.CosmosDB/src/CosmosDBConverter.cs#L58-L61

We should keep the pattern the same for consistency and if this is a behaviour we want to change we should open an issue to make this change to all converters.